app/views/categories/show.html.haml

```diff
@@ -22,5 +22,6 @@
   = render 'banner', position: :press_articles_top
   - if @category.has_children?
     = render 'categories', categories: @category.children, depth: 1
-  - else
-    = render 'posts', posts: @posts
+
+  %h2 このカテゴリの新着記事
```

Comment:
Where does this code come from?
I couldn't find in https://github.com/bm-sms/daimon-news/pull/582